### ISSUE TITLE
Level 6 features

### ISSUE THREAD

**Issue by efyphil** [philferriere/tfoptflow#21]
I tried to extract features and got very interesting results. 
![Screenshot from 2019-06-24 15-03-25](https://user-images.githubusercontent.com/38631588/60017474-79d5f480-9691-11e9-9cde-adb2080d5d11.png)
As you can see level 6 features was constant. I used different pictures and checked all of 196 feature maps and always got the same result. I also checked different weights for different realization of pwc-net. Can you please describe this result.

**Comment by lemon-PiC** [comment]
Did you find the answer? I have the same question in the 6th feature pyramid.

**Comment by efyphil** [comment]
> Did you find the answer? I have the same question in the 6th feature pyramid.

No, maybe you can try to learn network by yourself without last layer in encoder.

**Comment by Animadversio** [comment]
Doing some gradient analysis I found gradient vanishing phenomena to the last correlation layer or encoding layer is quite strong....
My intuition is the correlation output usually has a much smaller scale than the feature tensor, thus the activation could diminish, so do the gradient to the last decoder and encoder layer. 
(note that there is no direct connection from flow map to 6th layer feature tensor, all their gradient comes through the correlation layer, which seems like a correlation bottleneck )
